app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_per = [Person(body["name"], body["age"]) for body in people]
+    for human in people:
+        if human.get("wife") is not None:
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif human.get("husband") is not None:
+            Person.people[human["name"]].husband \
```

Comment:
Use `()` instead of `\`

Reply:
I can't do it, I tried but I got an error